Library/Formula/tasksh.rb

```diff
@@ -0,0 +1,19 @@
+class Tasksh < Formula
+  homepage "http://tasktools.org/projects/tasksh.html"
+  url "http://taskwarrior.org/download/tasksh-1.0.0.tar.gz"
+  head "https://git.tasktools.org/scm/ex/tasksh.git"
+  sha256 "9accc81f5ae3a985e33be728d56aba0401889d21f856cd94734d73c221bf8652"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/tasksh", "--version"
+    system "echo", "add", "Test Task", "|", "#{bin}/tasksh"
+    system "echo", "1", "del\nyes", "|", "#{bin}/tasksh"
```

Comment:
Sorry, this doesn't do what you think it does as it'll just echo the `|` rather than using a pipe. Instead use the `pipe_output` function and `assert_match` on the output.